LogosLinuxInstaller.py

```diff
@@ -297,13 +309,14 @@ def main():
     utils.die_if_root()
 
     # Print terminal banner
-    logging.info(f"{config.LOGOS_SCRIPT_TITLE}, {config.LOGOS_SCRIPT_VERSION} by {config.LOGOS_SCRIPT_AUTHOR}.")
+    logging.info(f"{config.LLI_TITLE}, {config.LLI_CURRENT_VERSION} by {config.LLI_AUTHOR}.")
     logging.debug(f"Installer log file: {config.LOGOS_LOG}")
 
-    utils.self_update()
+    utils.check_for_updates()
 
     # Check if app is installed.
-    install_not_required = [ 'remove_install_dir', 'check_dependencies', 'update_to_latest_recommended_appimage', 'set_appimage_symlink' ]
+    install_not_required = ['remove_install_dir', 'check_dependencies',  'update_self',
+                            'update_to_latest_recommended_appimage', 'set_appimage_symlink']
     if config.ACTION == "disabled":
         msg.logos_error("That option is disabled.", "info")
     elif config.ACTION.__name__ in install_not_required:  # These actions don't require an install
```

Comment:
The `update_self` action fails to get called here because in `install_not_required` it's listed with it's CLI argument "update_self", but the test on L322 checks the *function* name, which is actually "update_to_latest_lli_release". So either the function should be renamed to match the CLI argument, or the name given in `install_not_required` should be changed to match the function name.